yocto/docker/run-docker.sh

```diff
@@ -13,11 +13,20 @@ if [ "$2" ];then
 	EXTRA_ARGS="--volume $2:/tmp/sshagent --env=SSH_AUTH_SOCK=/tmp/sshagent"
 fi
 
+if [ ! "$3" ]; then
```

Comment:
make this optional, not every one uses this same for pki

Reply:
is optional now.